src/partials/header-content.hbs

```diff
@@ -13,7 +13,13 @@
         <a class="navbar-item" href="{{#with (and site.url site.homeUrl)}}{{{@root.site.url}}}{{{this}}}{{else}}{{{siteRootPath}}}{{/with}}">{{site.title}}</a>
         {{#if env.ALGOLIA_API_KEY}}
         <div class="navbar-item hide-for-print">
-          <input id="search-input" type="text" placeholder="Search docs">
+          <div class="search-field">
+            <input id="search-input" type="text" placeholder="Search docs">
```

Comment:
Should we update the placeholder?
If so, we could define another data attribute to declare the component title:

```hbs
<input type="checkbox" id="search-global"{{#if page.component}} data-component="{{page.component.name}}" data-component-title="{{page.component.title}}"{{/if}} checked>
```

And then use this value to update the placeholder:

```js
var componentTitle = document.getElementById('search-global').dataset.componentTitle
document.getElementById('search-input').placeholder = 'Search ' + componentTitle
```

Reply:
I think we should try to centralize on the page where the component name, version, and title are located. Right now, they are in the nav and explore. Let's keep it DRY. I'm open to ideas about where the best place for it is...though I'm thinking it should be the nav.

Reply:
> I think we should try to centralize on the page where the component name, version, and title are located.

I agree.

> I'm open to ideas about where the best place for it is...though I'm thinking it should be the nav.

I don't know, if we want the component name, version and title to be available everywhere we shouldn't declare them as data attribute on a UI component. The reason is that this component might not be present one every pages. For instance, on landing pages we might want to a different layout with an alternative navigation.

If we want them to be truly available on any pages then I think we should either declare them as `<meta>` or as data attributes on the `<body>`.

Reply:
I found a better way to do this by putting the full facet filter in the data attribute.